pkg/trace/test/testsuite/cards_test.go

```diff
@@ -22,12 +30,44 @@ func TestCreditCards(t *testing.T) {
 	}()
 
 	for _, tt := range []struct {
-		conf []byte
-		out  string
+		conf    []byte
+		out     string
+		version string
```

Comment:
Can you use `api.Version` ?
```suggestion
		version api.Version
```